packages/big-design-patterns/src/components/Header/styled.tsx

```diff
@@ -20,3 +21,10 @@ export const StyledBackLink = styled.a.attrs({ theme: defaultTheme })`
     outline: 4px solid ${({ theme }) => theme.colors.primary20};
   }
 `;
+
+export const StyledActionsWrapper = styled(Flex).attrs({ theme: defaultTheme })`
+  & > .bd-button + .bd-button {
+    margin-block-start ${({ theme }) => theme.spacing.none} !important;
+    margin-inline-start: ${({ theme }) => theme.spacing.none} !important;
+  }
+`;
```

Comment:
This overrides the code defined here: https://github.com/bigcommerce/big-design/blob/main/packages/big-design/src/components/Button/styled.tsx#L49-L57